### PR TITLE
OCPBUGS-35361: Updated the wait-ip note for bm only

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.13+

Issue:
[OCPBUGS-35361](https://issues.redhat.com/browse/OCPBUGS-35361)

Link to docs preview:
* [bare metal](https://77429--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#modifying-install-config-for-dual-stack-network_ipi-install-installation-workflow)
* [vsphere](https://77429--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.html#modifying-install-config-for-dual-stack-network_installing-vsphere-installer-provisioned-network-customizations)

- [x] SME has approved this change.
- [x] QE has approved this change.

### CODE REVIEW

**Review by cybertron — COMMENTED** [review]
/lgtm

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc [hunk 1]
```diff
@@ -28,9 +28,10 @@ serviceNetwork:
 - fd03::/112
 ----
 
+ifndef::vsphere[]
 [IMPORTANT]
 ====
-If you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, ensure you add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
+On a bare-metal platform, if you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, ensure you add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
```
Comment (mletalie) [inline modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc:34]:
```suggestion
On a bare-metal platform, if you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file. This  prevents your cluster from deploying on a dual-stack network.
```

Comment (mletalie) [reply modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc:34]: Just a suggestion, but I believe it removes unnecessary words as well as pointing out an "issue".

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/lgtm

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**